src/checksum/zseries/crc32-s390x.c

```diff
@@ -92,18 +93,6 @@ __wt_checksum_hw(const void *chunk, size_t len)
 }
 #endif
 
-extern uint32_t __wt_checksum_sw(const void *chunk, size_t len);
```

Comment:
Is it intentional to remove the externs

Reply:
Discussed offline, but for other reviewers - yes, we're now including `wiredtiger_internal.h` for `WT_MIN` so these are now duplicate declarations.